extensions/amp-analytics/0.1/cid-impl.js

```diff
@@ -422,10 +426,12 @@ function getEntropy(win) {
       win.Math.random() + win.screen.width + win.screen.height);
 }
 
+
 /**
- * @param {!Window} window
+ * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampdoc
  * @return {!Cid}
+ * @private visible for testing
  */
-export function installCidService(window) {
-  return fromClass(window, 'cid', Cid);
+export function installCidServiceForDoc(ampdoc) {
```

Comment:
We don't currently have such naming convention. Isn't cid service installed in [`amp-analytics`](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/amp-analytics.js#L43)? Could we use `installCidServiceForDoc` in `amp-analytics` instead of renaming?